### ISSUE TITLE
(1.7.0)cucumber export missing augments

### ISSUE THREAD

**Issue by FLemon** [hiptest/hiptest-publisher#65]
hi, on 1.7.0, exporting to cucumber-ruby
arguments are missing in the `*.feature` file
had to downgrade the version

**Comment by karamosky** [comment]
Hello @FLemon 
Could you provide more informations ? or maybe a diff between the last version and the latest one
Thanks

**Comment by FLemon** [comment]
sure, thought you can ran some test on your side to verify.
what I observed is as follow
exported using < 1.6.0
```
Feature: ExampleApp
  Scenario Outline: Example User login (uid: xxx)
    Given I am an app user "<user_role>"
    
    Examples:
      | user_role | hiptest-uid |
      | admin      | xxx              |
```

exported using 1.7.0
```
Feature: ExampleApp
  Scenario Outline: Example User login (uid: xxx)
    Given I am an app user
    
    Examples:
      | user_role | hiptest-uid |
      | admin      | xxx              |
```

**Comment by vincent-psarga** [comment]
Hi,

Do you use by any chance overriden-templates ? This might be the issue ?
Could you contact us via the in-app chat so we can try to reproduce your issues ?

Best regards,
Vincent

**Comment by FLemon** [comment]
@vincent-psarga not that i am aware any usage of overriden-templates
I can't see the in-app chat in your web app page anymore

**Comment by vincent-psarga** [comment]
Ok, I've send you an email, that should be easier :)

**Comment by cbliard** [comment]
Fixed in 1.7.1
